config/forms/category_details.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" ?>
+<form xmlns="http://schemas.sulu.io/template/template"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://schemas.sulu.io/template/template http://schemas.sulu.io/template/form-1.0.xsd"
+>
+    <key>category_details</key>
```

Comment:
Sulu automatically merges form-configurations that use the same key. This allows to hide existing properties and display new properties in existing forms. To use this mechanism for form-configurations with `<section>` tags, the workaround described in https://github.com/sulu/sulu/issues/5570 might be helpful.